src/IonReader.ts

```diff
@@ -136,6 +136,16 @@ export interface Reader {
    */
```

Comment:
Please add a `@deprecated` warning to the docs pointing users to the new `uInt8ArrayValue` method.